mp/src/game/client/neo/ui/neo_hud_ammo.cpp

```diff
@@ -101,6 +87,14 @@ void CNEOHud_Ammo::ApplySchemeSettings(vgui::IScheme* pScheme)
 {
 	BaseClass::ApplySchemeSettings(pScheme);
 
+	LoadControlSettings("scripts/HudLayout.res");
+
+	m_hSmallTextFont = pScheme->GetFont("NHudOCRSmall");
+	m_hBulletFont = pScheme->GetFont("NHudBullets");
+	m_hTextFont = pScheme->GetFont("NHudOCR");
+
+	InvalidateLayout();
```

Comment:
Nit: `LoadControlSettings` already calls `InvalidateLayout`, so we can probably omit this.